decidim-core/db/migrate/20220204121246_update_users_nickname.rb

```diff
@@ -10,31 +10,21 @@ def up
 
     Decidim::User.find_each do |user|
       next if has_changed.include? user
```

Comment:
Here I think we should follow the ad-hoc see [Data migration consistency](https://github.com/decidim/decidim/discussions/8068)